docs/versioning/olm.md

```diff
@@ -0,0 +1,98 @@
+## How to create a MGC OLM bundle, catalog and how to install MGC via OLM
+
+### Creating the bundle
+    :exclamation: NOTE: You can supply different env vars to the following make commands these include:
+    * Version using the env var VERSION 
+    * Tag via the env var IMAGE_TAG for tags not following the semantic format.
+    * Image registry via the env var REGISTRY
+    * Registry org via the env var ORG
+
+    For example
+
+    `make bundle VERISON=2.0.1`
```

Comment:
wondering if each version of the bundle should go into its own dir under manifests? To keep a history of the versions?

Reply:
not actually related to this code just something that came to mind